Assets/MixedRealityToolkit/_Core/Sharing/SimpleStatePlayerPrefs.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.Sharing;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Services.Sharing
+{
+    /// <summary>
+    /// A local simulation of state sharing
+    /// Uses JSON to store and retrieve serialized objects from PlayerPrefs
+    /// Useful for testing state sharing without a connection
+    /// </summary>
+    public class SimpleStatePlayerPrefs : BaseService, ISimpleState
+    {
+        public override void Initialize()
+        {
+            Name = "SimpleStatePlayerPrefs";
+        }
+
+        public int SimulatedLatency { get; set; }
```

Comment:
inheritdoc?

Same with all the other interface implementations so that the xml docs propagate down to the rest of the things that use this service.